homeassistant/components/light/hue.py

```diff
@@ -302,7 +302,7 @@ def __init__(self, light_id, info, bridge, update_lights,
     def unique_id(self):
         """Return the ID of this Hue light."""
         return "{}.{}".format(
-            self.__class__, self.info.get('uniqueid', self.name))
+            self.__class__, self.info.get('uniqueid', self.light_id))
```

Comment:
Questions on this: Are actual lights guaranteed to have 'uniqueid'? In other words, is the light_id only ever going to be used for groups?